pkg/cloudprovider/provider/vultr/types/types.go

```diff
@@ -22,11 +22,17 @@ import (
 )
 
 type RawConfig struct {
-	APIKey providerconfigtypes.ConfigVarString `json:"apiKey,omitempty"`
-	Region providerconfigtypes.ConfigVarString `json:"region"`
-	Plan   providerconfigtypes.ConfigVarString `json:"plan"`
-	OsID   providerconfigtypes.ConfigVarString `json:"osId"`
-	Tags   []string                            `json:"tags,omitempty"`
+	PhysicalMachine bool                                `json:"physicalMachine,omitempty"`
+	APIKey          providerconfigtypes.ConfigVarString `json:"apiKey,omitempty"`
+	Region          providerconfigtypes.ConfigVarString `json:"region"`
+	Plan            providerconfigtypes.ConfigVarString `json:"plan"`
+	OsID            providerconfigtypes.ConfigVarString `json:"osId"`
+	Tags            []string                            `json:"tags,omitempty"`
+	VpcID           []string                            `json:"vpcId,omitempty"`
+	Vpc2Id          []string                            `json:"vpc2Id,omitempty"`
```

Comment:
```suggestion
	Vpc2ID          []string                            `json:"vpc2Id,omitempty"`
```
Just to make sure it wasn't a miss - you seem to follow the uppercase notation for `ID` everywhere, but not in this case, is that intentional?

Reply:
No actually i prefer snakecase but the lint was failing due to that. So I made this partial change. Will make it consistent.